### PR TITLE
Add palmari recipe

### PR DESCRIPTION
**Pull request by goanpeca**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- ~[ ] If static libraries are linked in, the license of the static library is packaged.~
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by goanpeca** [comment]
Thanks @hippover!, Just accepting is enough. A side note, it seems the `tiffile` package is deprecated, so I would suggest updating the dependencies to use `tifffile`. (notice the extra `f`) for the next release of palmari.

Cheers!

**Comment by goanpeca** [comment]
This one is ready for review @conda-forge/staged-recipes

File: recipes/palmari/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+  run:
+    - dask-core ==2022.1.0
+    - dask-image ==2021.12.0
```
Comment (jaimergp) [inline recipes/palmari/meta.yaml:23]: @hippover, do you think we can be a bit more flexible here? Do newer dask versions work or was this the last one to work?

**Comment by hippover** [comment]
I haven’t tested but more recent versions should work as I do not remember deprecation warnings. We could replace the == by >=

**Pull request merged**